packages/zipkin/src/internalRepresentations.js

```diff
@@ -29,6 +30,17 @@ Endpoint.prototype.toJSON = function toJSON() {
   };
 };
 
+Endpoint.createEndpoint = function createEndpoint({serviceName, host, port}, callback) {
+  getAddressByName(host, addr => {
```

Comment:
I'd either leave the field as an ip addr (converting lazily) or rename the function to getIpV4. Remember we do actually support ipv6 and some won't have a ipv4 address, but this code seems to help in either case.

ps it seems like this should just affect the host field, not the entire endpoint.